### DISCUSSION TITLE
building desktop app from source

### DISCUSSION THREAD

**Discussion by brukg** [foxglove/community#757] [Q&A]
I cloned the repo to test it and after building it while yarn web:serve is working yarn desktop:start or yarn desktop:serve are not working. I can't seem to find any instructions in the docs. I have to mention a get many error during the build process for both web and desktop cases

**Comment by jtbandes** [comment]
We have some instructions here: https://github.com/foxglove/studio/blob/main/CONTRIBUTING.md
